### PR TITLE
Updates to Events and Calendar publishing rules

### PR DESCRIPTION
**Pull request by humb1t**

### CODE REVIEW

File: content/resources/rules.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Несмотря на наличие основных рубрик, мы с удовольствием поделимся интересной для подписчиков информацией, не подпадающей под вышеперечисленные категории.
+Любые публикации о некоммерческих мероприятиях, курсах и прочих событиях осуществляются на безвозмездной основе. Если же ваш анонс подразумевает коммерцию (платный вход, деньги за обучение и прочее), то мы опубликуем его лишь при выгоде для наших подписчиков. Это могут быть розыгрыш бесплатных мест на мероприятие или курсы, промокод со стоящей скидкой и тому подобное. Администраторы SITC не будут требовать вознаграждения для себя за публикацию каких-либо новостей, если вы столкнулись с подобной ситуацией — напишите об этом в ЛС Никите Пузанкову (@humb1t). Если же вы хотите отблагодарить сообщество, вы можете стать нашим [патроном](https://www.patreon.com/samara_it_community).
+
+### Мероприятия, календарь и анонсы
+
+Участники нашего сообщества, публикующие анонсы предстоящих мероприятий, могут добавить свои события в наш [календарь](https://sitc.community/events/) если:
+- Они явно попросят @humb1t забронировать под них какое-то время и дату;
```
Comment (gsomix) [inline content/resources/rules.md]:
```suggestion
- явно попросят @humb1t забронировать под них какое-то время и дату;
```

File: content/resources/rules.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Они сделают анонс мероприятия в котором явно укажут время, дату и место проведения; ссылку на регистрацию, если таковая требуется; ссылку на группу или сайт мероприятия.
```
Comment (gsomix) [inline content/resources/rules.md]:
```suggestion
- сделают анонс мероприятия, в котором явно укажут время, дату и место проведения; ссылку на регистрацию, если таковая требуется; ссылку на группу или сайт мероприятия.
```

**Comment by gsomix** [comment]
Спасибо!

**Comment by humb1t** [comment]
@gsomix thank you for the review ;-)

**Pull request merged**